CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ the endpoint has been removed. Use the `/metrics` endpoint to collect system sta
 1. [20827](https://github.com/influxdata/influxdb/pull/20827): Add `--pprof-disabled` option to `influxd` to disable exposing profiling information over HTTP.
 1. [20827](https://github.com/influxdata/influxdb/pull/20827): Add `/debug/pprof/all` HTTP endpoint to gather all profiles at once.
 1. [20827](https://github.com/influxdata/influxdb/pull/20827): Upgrade `http.pprof-enabled` config in `influxd upgrade`.
+1. [20911](https://github.com/influxdata/influxdb/pull/20911): Add support for explicitly setting shard-group durations on buckets. Thanks @hinst!
```

Comment:
Would it be more clear to call it "shard duration" everywhere? Some of the errors from the copied-over v1 code use that wording.

Reply:
So as I understand it:

`shard duration` == `shard group duration` is the amount of time each shard covers.

`duration` (of a retention policy) is the amount of time we actually keep the data for.

If we were starting over I'd want to call the first thing `shard span` or some other word, and the second thing the `duration`.

If we can't do that maybe we can standardize on 'shard group duration' and 'retention duration'?

@sanderson might have a recommendation